### PR TITLE
doc: Add limitations section to top level readme

### PR DESCRIPTION
**Pull request by zach-carr**
Documentation update - [DOCSP-29472](https://jira.mongodb.org/browse/DOCSP-29472)
- Adds a limitations section

A markdown viewer I found online, if it's helpful for copy review: https://markdownlivepreview.com/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,16 @@ See the [`cfn` examples](examples/README.md) to setup prerequisites and get star
 ### MongoDB Atlas AWS CDK
 See the [cdk examples](https://github.com/mongodb/awscdk-resources-mongodbatlas/blob/main/examples/README.md) to setup prerequisites and get started with your first cluster using our AWS CDK sample code.
 
+## Limitations
+1. [Resource import](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resource-import.html) is not supported for third-party resources.
+
+2. CloudFormation support for any third-party resource requires that the third-party resource support all CRUD operations. For this reason, the following resources are not supported:
```
Comment (sarahsimpers) [inline README.md:19]: Optional suggestion, this sentence is hard to parse/understand, if this conveys the same meaning you were going for it may be clearer:
```suggestion
2. The following resources are not supported because they require support for all CRUD operations: 
```

**Review by sarahsimpers — APPROVED** [review]
Approved with one suggestion to make the meaning of 2 more clear

File: README.md (hunk 1 above)
Comment (zach-carr) [reply README.md:19]: Good call, this is hard to parse. I want to communicate the general requirement that any third party resource would need to support all CRUD ops to work because these third-party tools change without our knowledge. I'm thinking:

"Any third-party resource you integrate must support all CRUD operations. CloudFormation does not support the following resources because they do not support all CRUD operations:"

File: README.md [hunk 2]
```diff
@@ -13,6 +13,16 @@ See the [`cfn` examples](examples/README.md) to setup prerequisites and get star
 ### MongoDB Atlas AWS CDK
 See the [cdk examples](https://github.com/mongodb/awscdk-resources-mongodbatlas/blob/main/examples/README.md) to setup prerequisites and get started with your first cluster using our AWS CDK sample code.
 
+## Limitations
+1. [Resource import](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resource-import.html) is not supported for third-party resources.
+
+2. Any third-party resource you integrate must support all CRUD operations. CloudFormation does not support the following resources because they do not support all CRUD operations:
```
Comment (andreaangiolillo) [inline README.md:19]:
```suggestion
2. Any third-party resource must support all CRUD operations. MongoDB Atlas for CloudFormation does not support the following MongoDB Atlas resources because they do not support all CRUD operations: 
```

**Review by andreaangiolillo — COMMENTED** [review]
LGTM. My comment is not blocking

File: README.md (hunk 2 above)
Comment (zach-carr) [reply README.md:19]: This is clearer, thanks

**Review by andreaangiolillo — APPROVED** [review]

**Comment by zach-carr** [comment]
Thanks for the review @andreaangiolillo  - would you mind taking a look at this identical PR? https://github.com/mongodb/awscdk-resources-mongodbatlas/pull/116
